data/remnant outfits.txt

```diff
@@ -341,6 +341,23 @@ outfit "Quantum Key Stone"
 
 
 
+# Scanner:
+
+outfit "Salvage Scanner"
+	category "Systems"
+	licenses
+		Remnant
+	cost 100000
```

Comment:
This is cheaper than a tactical scanner is. I think something around say 390k is more appropriate.

Reply:
Sorry. That was a holdover from the Software Scanner. 390k would be good, but I think 410k would be better. That might help emphasize the fact that this a specialized piece of gear, not some generic scanner that is being mass produced for large swaths of the human market.